Tests/Contour.Common.Tests/PatternsSpecs.cs

```diff
@@ -287,7 +287,7 @@ public void should_accept_message()
                     "Body");
 
                 Mock<IDelivery> deliveryMock = new Mock<IDelivery>();
-                BusProcessingContext.Current = new BusProcessingContext(deliveryMock.Object);
+                BusProcessingContext.Current = new BusProcessingContext(deliveryMock.Object, null);
```

Comment:
As long as instances under test don't need it I see no reason to add mocks as long as null suffice.